omnigibson/scene_graphs/graph_builder.py

```diff
@@ -0,0 +1,272 @@
+import itertools
+import os
+
+import networkx as nx
+import numpy as np
+from PIL import Image
+from matplotlib import pyplot as plt
+
+from omnigibson import object_states
+from omnigibson.sensors import VisionSensor
+from omnigibson.object_states.factory import get_state_name
+from omnigibson.object_states.object_state_base import AbsoluteObjectState, BooleanStateMixin, RelativeObjectState
+from omnigibson.utils import transform_utils as T
+
+DRAW_EVERY = 1
+
+
+def get_robot(scene):
```

Comment:
we have a lot of these "get_robot" APIs distributed throughout the code. can we unify to one single API? and maybe bake it into the sim class or something more centralized?